### PR TITLE
Set the codecA and codecV always to lover case

### PR DESCRIPTION
**Pull request by valib**
This change is made to avoid using containsIgnoreCase() and equalsIgnoreCase(). This could speed up the code.

### CODE REVIEW

**Comment by Sami32** [comment]
@valib Lokk very good to me...except for isAC3 and isDTS lines.
Why not used `contains` and reduced the "elements" to 2 and not 3 as now ?

File: src/main/java/net/pms/dlna/DLNAMediaAudio.java [hunk 1]
```diff
@@ -145,21 +145,21 @@ public int getSampleRate() {
 	 * @return True if the audio codec is AC-3.
 	 */
 	public boolean isAC3() {
-		return getCodecA() != null && (getCodecA().equalsIgnoreCase("ac3") || getCodecA().equalsIgnoreCase("a52") || getCodecA().equalsIgnoreCase("liba52"));
+		return getCodecA() != null && ("ac3".equals(getCodecA()) || "a52".equals(getCodecA()) || "liba52".equals(getCodecA()));
```
Comment (Sami32) [inline src/main/java/net/pms/dlna/DLNAMediaAudio.java]: `return getCodecA() != null && ("ac3".equals(getCodecA()) || "a52".contains(getCodecA()));`

File: src/main/java/net/pms/dlna/DLNAMediaAudio.java [hunk 2]
```diff
@@ -145,21 +145,21 @@ public int getSampleRate() {
...
-		return getCodecA() != null && getCodecA().equalsIgnoreCase("truehd");
+		return getCodecA() != null && "truehd".equals(getCodecA());
 	}
 
 	/**
 	 * @return True if the audio codec is DTS.
 	 */
 	public boolean isDTS() {
-		return getCodecA() != null && (getCodecA().startsWith("dts") || getCodecA().equalsIgnoreCase("dca") || getCodecA().equalsIgnoreCase("dca (dts)"));
+		return getCodecA() != null && (getCodecA().startsWith("dts") || "dca".equals(getCodecA()) || "dca (dts)".equals(getCodecA()));
```
Comment (Sami32) [inline src/main/java/net/pms/dlna/DLNAMediaAudio.java]: `return getCodecA() != null && ("dts".contains(getCodecA()) || "dca".contains(getCodecA()));`

**Comment by valib** [comment]
@Sami32 why to use contains for isAC3 and isDTS? I used the original code and only changed the equalsIgnoreCase to equals.

**Comment by Sami32** [comment]
@valib For code simplification.

**Comment by valib** [comment]
@Sami32 ahh I see. Good idea.

**Pull request merged**